app/views/application/_menu.html.erb

```diff
@@ -14,7 +14,7 @@
         <li><%= link_to "Menu", categories_path, :method => :get %></li>
       <% elsif !current_user.admin? %>
         <li class="dropdown">
-          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Profile <span class="caret"></span></a>
+          <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false" data-no-turbolink>Profile <span class="caret"></span></a>
```

Comment:
We'll have to come back and see if something other than turbolinks is causing the lag